frontend/src/pages/MilestonesEditor.module.scss

```
@@ -102,3 +104,20 @@
     padding-bottom: 0;
   }
 }
+
+.topbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border-bottom: 1px solid $COLOR_BLACK40;
+  margin-left: 23px;
+  padding: 10px 0px;
+}
+
+.topbarShares {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 0px 10px;
+}
```

Titles could have the pre-title font here or you can add class to the div's of the titles. Also some gap between the elements look good.

```suggestion
.topbarShares {
  @extend .typography-pre-title;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  gap: 5px;
  margin: 0px 10px;
}
```

---

`display: flex` and `flex-direction: column;` can be changed to `@extend .layout-column;` here as well